frontend-instructor/src/components/footer/Footer.jsx

```
@@ -1,71 +1,79 @@
             <svg
-              xmlns="http://www.w3.org/2000/svg"
-              width="40"
-              height="40"
-              viewBox="0 0 48 48"
-              className="transition-transform transform hover:scale-110"
+              xmlns='http://www.w3.org/2000/svg'
+              width='40'
+              height='40'
+              viewBox='0 0 48 48'
+              className='transition-transform transform hover:scale-110'
             >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)